src/components/views/rooms/EditMessageComposer.js

```diff
@@ -240,6 +246,16 @@ export default class EditMessageComposer extends React.Component {
         return caretPosition;
     }
 
+    _onChange = () => {
+        if (this.state.changed || !this._editorRef || !this._editorRef.isModified()) {
+            return;
+        }
+
+        this.setState({
+            changed: true,
```

Comment:
Nit: maybe we can call this `saveDisabled` as I can imagine people starting to use `this.state.changed` instead of `this._editorRef.isModified()` in the future to check for modifications.

Reply:
fair enough